generator/transit_generator.cpp

```diff
@@ -319,7 +208,179 @@ void BuildTransit(string const & mwmDir, string const & countryId,
   w.Seek(startOffset);
   header.Visit(serializer);
   w.Seek(endOffset);
+
   LOG(LINFO, (TRANSIT_FILE_TAG, "section is ready. Header:", header));
 }
+
+void GraphData::Append(GraphData const & rhs)
+{
+  m_stops.insert(m_stops.begin(), rhs.m_stops.begin(), rhs.m_stops.end());
```

Comment:
Looks like you're doing not Append, but Prepend. Moreover, could you please implement following helper:
```c++
template <typename T>
void Append(vector<T> & src, vector<T> const & dst)
{
  src.insert(src.end(), dst.begin(), dst.end());
}
```
and use it? My eyes are in pain from all these inserts, begins and ends.

Reply:
Done, unfortunately I cannot use visitor here because of its param.